src/Symfony/Component/ErrorHandler/ErrorHandler.php

```diff
@@ -442,7 +437,7 @@ public function handleError($type, $message, $file, $line)
                 self::$silencedErrorCache[$id][$message] = $errorAsException;
             }
             if (null === $lightTrace) {
-                return;
+                return null;
```

Comment:
this should not return null but true, see https://github.com/symfony/symfony/pull/32143/files#diff-e7d086f2c2086fdb8f108e74f11dac5b